feat(changelog): jobs-fixed-cancel-queued-jobs 2025-01-14 (#4218)
scaleway/docs-content: changelog/january2025/2025-01-14-jobs-fixed-cancel-queued-jobs.mdx

---

> title: Cancel queued Jobs

Reviewer: ```suggestion
title: Cancel queued jobs
```

status: fixed
author:
    fullname: 'Join the #serverless-jobs channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2025-01-14
category: serverless
product: jobs
---

> Sometimes Job Run in **queued** status couldn't be canceled, this is now fixed.

Reviewer 2: ```suggestion
Sometimes Job runs in **queued** status could not be effectively canceled. This issue is now fixed.
```

> Sometimes Job runs in **queued** status could not be effectively canceled. This issue is now fixed.

Reviewer: ```suggestion
Sometimes job runs in **queued** status could not be effectively canceled. This issue is now fixed.
```
